README.md

```diff
@@ -1,5 +1,26 @@
+# Windows Package Manager Repository
 
-# Contributing
+## Create a new private repository in Azure
+To simply the creation of a Windows Package Manager private repository, the `automation.ps1` [PowerShell script](Tools/Automation.ps1) is offered. This script will create new Elements required to host a Windows Package Manager private repository in Azure.
```

Comment:
```suggestion
To simplify the creation of a Windows Package Manager private source, the `automation.ps1` [PowerShell script](Tools/Automation.ps1) is offered. This script will create new resources required to host a Windows Package Manager private source in Azure.
```
Suggest aligning with azure resource names.

Are we aligning on the name 'repository' for any collection of apps even if it is not a github repository? Or is this a source which contains apps?

Reply:
I confirmed with Demitrius and we are aligning with "source". I'll update the documents